crates/node-file-trace/src/lib.rs

```diff
@@ -287,6 +293,12 @@ async fn input_to_modules<'a>(
             ..Default::default()
         }
         .cell(),
+    };
+    let context: AssetContextVc = ModuleAssetContextVc::new(
+        TransitionsByNameVc::cell(HashMap::new()),
```

Comment:
This cell call will make every call of input_to_modules to have a separate AssetContext, so multiple call will not share modules.
Could you move the AssetContext creation into a separate TT function